### PR TITLE
Edit docs/learn/about-iota/an-introduction-to-iota.md

### PR DESCRIPTION
**Pull request by DafPhil**

### CODE REVIEW

**Review by DafPhil — COMMENTED** [review]
Just proof-reading these pages on behalf of IOTA

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 1]
```diff
@@ -1,87 +1,73 @@
...
+- IOTA, in contrast, is a leaderless protocol. Everyone is free to attach transactions as long as they are following the basic layout design of the protocol. There is no need to elect a leader as the Tangle can implement new transactions in parallel processing. And therefore, IOTA removed the middlemen entirely from the protocol. This is one of the reasons why IOTA can reach a very high transaction throughput and also remains absolutely feeless.
 
 To explain these points, you need to understand the differences between the data structures and the consensus mechanisms in these DLTs.
 
 **Blockchain vs. Tangle**
 
-[![layer1-tangle](/img/learn/layer1-tangle.png)](/img/learn/layer1-tangle.png)
+![layer1-tangle](/img/learn/layer1-tangle.png)
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:25]:
```suggestion
[![layer1-tangle](/img/learn/layer1-tangle.png)](/img/learn/layer1-tangle.png)
```

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 2]
```diff
@@ -1,87 +1,73 @@
...
+The blockchain data structure consists of a chain of sequential blocks, where each block contains a limited number of messages. As a result, Validators can attach new messages to only one place: a block at the end of the chain. Due to this limitation, blockchain networks often experience slow confirmation times. This limitation is known as the blockchain bottleneck.
 
 **Blockchain bottleneck**
 
-[![blockchain-bottleneck](/img/learn/blockchain-bottleneck.gif)](/img/learn/blockchain-bottleneck.gif)
-
-All transactions in a Blockchain have to wait until they are included in a Block. Due to limitations in Blocksize and Block Production time, this creates congestion and waiting times for transactions.
-
-
+![blockchain-bottleneck](/img/learn/blockchain-bottleneck.gif)
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:33]:
```suggestion
[![blockchain-bottleneck](/img/learn/blockchain-bottleneck.gif)](/img/learn/blockchain-bottleneck.gif)
```

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 3]
```diff
@@ -1,87 +1,73 @@
...
 ## **The Tangle data structure**
 
-The Tangle data structure is a directed acyclic graph (DAG), where each message is attached to two to eight previous ones.
-Rather than being limited to a single place for attaching new messages, you can attach messages in different places at the front of the Tangle. The protocol is able to process these various attachments in parallel.
+The Tangle data structure is a directed acyclic graph (DAG), where each message is attached to two to eight previous ones. Rather than being limited to a single place for attaching new messages, you can attach messages in different places at the front of the Tangle. The protocol is able to process these various attachments in parallel.
 
 **Tangle bottleneck**
 
-[![tangle-bottleneck](/img/learn/tangle-bottleneck.gif)](/img/learn/tangle-bottleneck.gif)
+![tangle-bottleneck](/img/learn/tangle-bottleneck.gif)
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:43]:
```suggestion
[![tangle-bottleneck](/img/learn/tangle-bottleneck.gif)](/img/learn/tangle-bottleneck.gif)
```

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 4]
```diff
@@ -1,87 +1,73 @@
...
 
 Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
 
-To allow the nodes recognizing the milestones, all nodes that participate in the same IOTA network are configured with the Merkle root address of a Coordinator that they trust to confirm messages. Using this address, nodes can validate the signatures in milestones to verify whether the trusted Coordinator signs them.
+To allow the nodes to recognize the milestones, all nodes that participate in the same IOTA network are configured with the Merkle root address of a Coordinator that they trust to confirm messages. Using this address, nodes can validate the signatures in milestones to verify whether the trusted Coordinator signs them.
 
 To make sure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones at regular intervals. This way, nodes can compare the indexes of their milestones to check whether they are synchronized with the rest of the network.
 
-[![coordinator](/img/learn/milestones.gif)](/img/learn/milestones.gif)
+![coordinator](/img/learn/milestones.gif)
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:73]:
```suggestion
[![coordinator](/img/learn/milestones.gif)](/img/learn/milestones.gif)
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Review by Dr-Electron — APPROVED** [review]

**Comment by Dr-Electron** [comment]
Really appreciate your contribution ❤️

**Pull request merged**